dist/linux/cpeditor.desktop

```diff
@@ -14,3 +14,5 @@ Terminal=false
 StartupNotify=true
 Encoding=UTF-8
 Categories=Development;TextEditor;Qt;
+MimeType=text/plain;text/x-c++hdr;text/x-c++src;text/x-chdr;text/x-csrc;text/x-java;text/x-c;text/x-c++;
```

Comment:
```suggestion
MimeType=text/x-c++hdr;text/x-c++src;text/x-chdr;text/x-csrc;text/x-java;text/x-python
```

Reply:
Some OSs may mark code files as `text/plain`.

Reply:
> Some OSs may mark code files as `text/plain`.

Can you give a reference? And I also want to know where did you find `text/x-c` and `text/x-c++` :thinking: